.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+env:
+  - DJANGO_SETTINGS_MODULE=concordia.settings_test
+python:
+  - "3.6"
+  - "3.6-dev"  # 3.6 development branch
+  - "3.7-dev"  # 3.7 development branch
+# command to install dependencies
+install:
+  - pip install pipenv
+  - pipenv install --dev --deploy
+# command to run tests
+script:
+  - mkdir logs
+  - touch ./logs/concordia-celery.log
```

Comment:
Based on the failures, we probably need to add `./manage.py collectstatic --noinput` here, too

Reply:
https://github.com/LibraryOfCongress/concordia/pull/317/commits/e4529233a5f08d401bf24017d1fc019da8a11845

Reply:
I tested with the mkdir and touch commands removed (after collectstatic was added) and I had to add them back in.